accepted/targeting-packs/targeting-packs.md

```diff
@@ -0,0 +1,159 @@
+  `TargetFrameworkRootPath` to a path within the NuGet package. This will allow
+  the existing MSBuild logic in the `GetReferenceAssemblyPaths` target and other
+  logic such as automatically referencing the facades if necessary to continue
+  to work as normal.
+    - We may need to set `FrameworkPathOverride` as well to avoid other weird
+      issues like the explicit `mscorlib` reference.
+* The logic in the `.targets` file in the NuGet packages will only set the
+  `TargetFrameworkRootPath` if the `TargetFrameworkIdentifier` and
+  `TargetFrameworkVersion` properties match the reference assemblies that the
+  package provides.
+* The reference assembly packages should include the `.xml` intellisense
+  documentation files. I believe the `.xml` documentation files have to be in
+  the same folder as the reference assemblies, so there isn't a way to deliver
+  them as a separate package.
+* If we want to support localized intellisense, we would need to create a
+  separate set of packages and corresponding meta-package for each language
+  supported. The IDs of these packages could follow the pattern
+  `Microsoft.NETFramework.ReferenceAssemblies.pt-br`. The SDK could use a
+  property to select which language's metapackage to reference.
+* The reference assembly packages should not show up as dependencies of "normal"
+  packages. Thus, the reference assembly packages should set
+  `developmentDependency` to true in it's metadata. Likewise, when the .NET SDK
+  automatically references the reference assembly metapackage, it should use
+  `PrivateAssets="All"`.
+* The reference assembly packages should include the same layout of files that
+  are installed by the targeting packs under `C:\Program Files (x86)\Reference
+  Assemblies\Microsoft\Framework`. This should be rooted at the path specified
+  by the `TargetFrameworkRootPath` property in the package. For example, if the
+  `.targets` file, which is in the `build` folder of the NuGet package, sets the
+  `TargetFrameworkRootPath` to `$(MSBuildThisFileDirectory)`, then the .NET
+  4.6.2 reference assembly package should have the reference assemblies and
+  intellisense files in the `build\.NETFramework\v4.6.2` folder, as well as have
+  the `Facades`, `PermissionSets`, and `RedistList` folders with corresponding
+  files under that folder.
+* The version number of each package should start at 1.0.0. When a new version
+  of the .NET Framework is released, a corresponding reference assembly package
+  (versioned at 1.0.0) should be released, and a new metapackage that includes
+  the additional dependency for the newly supported version should be released.
+  The new version of the metapackage should have it's minor version incremented.
+  If we need to fix an issue with the packages, we can ship new versions with
+  the patch version incremented, and a new metapackage with dependencies on the
+  patched packages.
+* We need to determine what license to use for these packages.
```

Comment:
Is there any reason not to use an open source license, like MIT, for these reference assemblies?